### DISCUSSION TITLE
is it possible to add google pay button  + apple pay buttons separate to card field?

### DISCUSSION THREAD

**Discussion by developermvuk** [flutter-stripe/flutter_stripe#984] [Q&A]
Hi, 

I was wondering if it's possible to add google pay button  + apple pay buttons separate to card field without the buttons linked to the credit card field section? Would appreciate any assistance on this.

**Comment by developermvuk** [comment]
If anyone can help will be appreciated? I just to know if we can separate the Apple pay button + google pay button from the card payment sheet?

**Comment by remonh87** [comment]
no this is not possible. The paymentsheet is part of prebuild UI by Stripe. However most of the time users have either google pay or apple pay so I think there won't be a big problem.
